app/main/views/services.py

```diff
@@ -59,14 +70,17 @@ def edit_service(service_id):
     )
 
 
-@main.route('/services/<string:service_id>/remove', methods=['POST'])
+@main.route("/frameworks/<string:framework_slug>/services/<string:service_id>/remove", methods=['POST'])
 @login_required
-def remove_service(service_id):
+def remove_service(framework_slug, service_id):
     service = data_api_client.get_service(service_id).get('services')
```

Comment:
In the `edit` and `update` views below, we check that the service exists here before getting `services`.
```
service = data_api_client.get_service(service_id)
if service is None:
    abort(404)
service = service['services']
```
Is that something we should do here?